torch/csrc/jit/script/compiler.cpp

```diff
@@ -1248,7 +1266,13 @@ struct to_ir {
     if (it != function_table.end()) {
       return std::make_shared<SimpleValue>(packOutputs(*graph, method.emit_call_to(ident.range(), it->second, inputs, attributes)));
     }
-    if(auto result = emitBuiltinCall(ident.range(), *method.graph(), Symbol::aten(ident.name()), inputs, attributes, false)) {
+    if (auto result = emitBuiltinCall(
+            ident.range(),
+            *method.graph(),
+            {Symbol::aten(ident.name()), Symbol::prim(ident.name())},
```

Comment:
Hmm it would be better if we could choose the namespace based on the type of the argument. So lists would always go to `prim`, but tensors would always go to `aten`.

Reply:
I feel like that would add even more complexity to the compiler. Schema matching already chooses the right operator based on the type of the arguments. I guess we would save some time when looking for ops to match against but it seems unnecessary?